src/vs/workbench/contrib/notebook/browser/view/notebookCellList.ts

```diff
@@ -1205,10 +1205,6 @@ export class NotebookCellList extends WorkbenchList<CellViewModel> implements ID
 		const focused = this.getFocus();
 		const focus = focused.length ? focused[0] : null;
 
-		if (this.view.elementTop(index) >= this.view.getScrollTop()) {
```

Comment:
this check is part of the `scrollHeuristic` below should not determine alone where to anchor